### PR TITLE
feat: support Casbin BatchAdapter interface

### PR DESCRIPTION
**Pull request by basakest**

### CODE REVIEW

File: src/Adapters/DatabaseAdapter.php [hunk 1]
```diff
@@ -145,4 +146,51 @@ public function removeFilteredPolicy(string $sec, string $ptype, int $fieldIndex
...
+    public function removePolicies(string $sec, string $ptype, array $rules): void
+    {
+        $ids = [];
+        foreach($rules as $rule) {
+            $where = [];
+            $where['ptype'] = $ptype;
+            foreach ($rule as $key => $value) {
+                $where['v'.strval($key)] = $value;
+            }
+            $ids[] = RulesModel::create()->get($where)->id;
```
Comment (Player626) [inline src/Adapters/DatabaseAdapter.php]: Change it to the following code:
```php
$ret = RulesModel::create()->get($where);
$ret && $ids[] = $ret->id;
```

Comment (hsluoyz) [reply src/Adapters/DatabaseAdapter.php]: @basakest is it resolved?

**Comment by hsluoyz** [comment]
@techoner plz review.

File: src/Adapters/DatabaseAdapter.php (hunk 1 above)
Comment (Player626) [reply src/Adapters/DatabaseAdapter.php]: @hsluoyz yes

**Comment by Player626** [comment]
The merger can be carried out.

File: src/Adapters/DatabaseAdapter.php (hunk 1 above)
Comment (hsluoyz) [reply src/Adapters/DatabaseAdapter.php]: Then click 
image

**Pull request merged**